modules/odc-importing-codebase-from-git-to-create-application.adoc

```diff
@@ -21,10 +21,11 @@ The following procedure walks you through the *From Git* option in the *Develope
 * *Context Dir* to specify the subdirectory for the application source code you want to use to build the application.
 * *Source Secret* to create a *Secret Name* with credentials for pulling your source code from a private repository.
 
-. Optional: You can import a `Devfile`, a `Dockerfile`, or a `Builder Image` through your Git repository to further customize your deployment.
-* If your Git repository contains a `Devfile`, a `Dockerfile`, or a `Builder Image`, it is automatically detected and populated on the respective path fields. If a `Devfile`, a `Dockerfile`, or a `Builder Image` are detected in the same repository, the `Devfile` is selected by default.
+. Optional: You can import a `Devfile`, a `Dockerfile`, `Builder Image` or a `Serverless Function` through your Git repository to further customize your deployment.
+* If your Git repository contains a `Devfile`, a `Dockerfile`, `Builder Image`, or a `Serverless Function`, it is automatically detected and populated on the respective path fields. If a `Devfile`, a `Dockerfile`, or a `Builder Image` are detected in the same repository, the `Devfile` is selected by default. f `func.yaml` is detected in the Git repository, the *Import Strategy* changes to `Serverless Function`.
```

Comment:
@vikram-raj I think this should be changed to the following, WDYT?
```suggestion
* If your Git repository contains a `Devfile`, a `Dockerfile`, `Builder Image`, or a `Serverless Function YAML`, it is automatically detected and populated on the respective path fields. If a `Devfile`, a `Dockerfile`, or a `Builder Image` are detected in the same repository, the `Devfile` is selected by default. If `func.yaml` is detected in the Git repository, the *Import Strategy* changes to `Serverless Function`.
```

Reply:
We can simply say `a func.yaml` instead of `Serverless Function`.

Reply:
Made the changes. Thanks!